### PR TITLE
[Docs] add info about subresources

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.11
| Bug fix?        | no                                                       |
| New feature?    | no                                                     |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets |                       |
| License         | MIT                                                          |

image

### CODE REVIEW

File: docs/customization/api/customizing_endpoints_using_yaml.rst [hunk 1]
```diff
@@ -59,3 +59,28 @@ You can also overwrite existing endpoints, for example let's change admin_get op
                     groups: ['shop:channel:new_group']
 
 This way we can edit the existing endpoint and add custom normalization or change path.
+
+.. warning::
+
+    By removing subresource operations, api platform creates its own endpoints to given subresoure without our firewall.
```
Comment (loic425) [inline docs/customization/api/customizing_endpoints_using_yaml.rst:65]:
```suggestion
    By removing subresource operations, api platform creates its own endpoints to given subresource without our firewall.
```

**Review by loic425 — DISMISSED** [review]

**Review by AdamKasp — APPROVED** [review]

File: docs/customization/api/customizing_endpoints_using_yaml.rst [hunk 2]
```diff
@@ -59,3 +59,28 @@ You can also overwrite existing endpoints, for example let's change admin_get op
                     groups: ['shop:channel:new_group']
 
 This way we can edit the existing endpoint and add custom normalization or change path.
+
+.. warning::
+
+    By removing subresource operations, api platform creates its own endpoints to given subresource without our firewall.
```
Comment (GSadee) [inline docs/customization/api/customizing_endpoints_using_yaml.rst:65]:
```suggestion
    By removing subresource operations, API Platform creates its own endpoints to a given subresource without our firewall.
```

**Review by GSadee — COMMENTED** [review]
And this PR should be opened to 1.11 branch

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "disable-subresource-docs"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, @SirDomin! :1st_place_medal:

**Pull request merged**